.github/workflows/ci.yml

```diff
@@ -429,45 +429,44 @@ jobs:
         run: |
           ./build_tools/cmake/ctest_all.sh ${BUILD_DIR}
 
-  # TODO(#17370): re-enable when nodai-amdgpu-w7900-x86-64 runner is stable
-  # test_amd_w7900:
-  #   needs: [setup, build_all]
-  #   if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'test_amd_w7900')
-  #   env:
-  #     BUILD_DIR: build-tests
-  #     INSTALL_DIR: ${{ needs.build_all.outputs.install-dir }}
-  #     INSTALL_DIR_ARCHIVE: ${{ needs.build_all.outputs.install-dir-archive }}
-  #     INSTALL_DIR_GCS_URL: ${{ needs.build_all.outputs.install-dir-gcs-url }}
-  #     IREE_CPU_DISABLE: 1
-  #     IREE_VULKAN_DISABLE: 0
-  #     IREE_CUDA_DISABLE: 1
-  #     IREE_HIP_DISABLE: 0
-  #     IREE_HIP_TEST_TARGET_CHIP: "gfx1100"
-  #   runs-on: nodai-amdgpu-w7900-x86-64
-  #   steps:
-  #     - name: "Checking out repository"
-  #       uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
-  #     - name: "Checking out runtime submodules"
-  #       run: ./build_tools/scripts/git/update_runtime_submodules.sh
-  #     - name: "Downloading install dir archive"
-  #       run: wget "${INSTALL_DIR_GCS_URL}" -O "${INSTALL_DIR_ARCHIVE}"
-  #     - name: "Extracting install directory"
-  #       run: tar -xf "${INSTALL_DIR_ARCHIVE}"
-  #     - name: "Building tests"
-  #       run: |
-  #         ./build_tools/pkgci/build_tests_using_package.sh ${INSTALL_DIR}
-  #     - name: "Running GPU tests"
-  #       env:
-  #         IREE_CTEST_LABEL_REGEX: ^requires-gpu|^driver=vulkan$|^driver=hip$
-  #         IREE_AMD_RDNA3_TESTS_DISABLE: 0
-  #         IREE_NVIDIA_GPU_TESTS_DISABLE: 0
-  #         IREE_NVIDIA_SM80_TESTS_DISABLE: 1
-  #         IREE_MULTI_DEVICE_TESTS_DISABLE: 0
-  #         IREE_CUDA_DISABLE: 1
-  #         IREE_CPU_DISABLE: 1
-  #         IREE_HIP_DISABLE: 0
-  #       run: |
-  #         ./build_tools/cmake/ctest_all.sh ${BUILD_DIR}
+  test_amd_w7900:
+    needs: [setup, build_all]
+    if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'test_amd_w7900')
+    env:
+      BUILD_DIR: build-tests
+      INSTALL_DIR: ${{ needs.build_all.outputs.install-dir }}
+      INSTALL_DIR_ARCHIVE: ${{ needs.build_all.outputs.install-dir-archive }}
+      INSTALL_DIR_GCS_URL: ${{ needs.build_all.outputs.install-dir-gcs-url }}
+      IREE_CPU_DISABLE: 1
+      IREE_VULKAN_DISABLE: 0
+      IREE_CUDA_DISABLE: 1
+      IREE_HIP_DISABLE: 0
+      IREE_HIP_TEST_TARGET_CHIP: "gfx1100"
+    runs-on: nodai-amdgpu-w7900-x86-64
```

Comment:
When will we have more than one runner? Even when the runner was relatively stable, we still saw long queue times with how many jobs there are now.

Reply:
In the next couple of days (fingers crossed). pushing on getting that done

Reply:
:/ yeah, already seeing some 45+ minute queues.